README.md

```diff
@@ -37,21 +37,47 @@ The code organisation is described [here](doc/CodeOrganization.md).
 The build system (cmake) is described [here](doc/CMakeInstructions.md).
 
 ### Formatting
-##### Using clang-format
+The project uses `clang-format` to push for a common code formatting according
+the the `clang-format` configuration files in this repository. With an adiabatic
+approach, all changes have to follow the formatting rules. A tool script can be
+used to integrate the formatting into `git` and suggest formatting only for
+changed lines.
+
+##### Install `clang-format` git integration
```

Comment:
Why not simply including it in O2?

Reply:
true, however not sure if it belongs there

Reply:
IMHO, telling people to wget files is not what we want to do. What we support is either "alibuild" or standard "brew install" / "apt-get install". We should avoid proliferation of installation mechanism for different tools.

Reply:
@dberzano

Reply:
I fully agree with @ktf. We can possibly have the script distributed with O2 in the utilities folder.